setup.py

```diff
@@ -7,8 +7,18 @@ def read(fname):
         return file.read()
 
 
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith('__version__'):
+            delim = '"' if '"' in line else "'"
+            return line.split(delim)[1]
+    else:
+        raise RuntimeError("Unable to find version string.")
+
+
 setuptools.setup(
     name='epcsunspecdemo',
+    version=get_version('./src/epcsunspecdemo/_version.py'),
```

Comment:
If we switch over to a `setup.cfg` (in another PR, which would be good.  see #7) then I think we can make setuptools take care of this for us.

https://github.com/altendky/qtrio/blob/f8ef0a86d41b15f0b189edf888a204b7c9d6d35a/setup.cfg#L3
```python
version = attr: qtrio._version.__version__
```

Though hmm...  what about the `src/` bit.  Does it handle that?  Welp, we'll have to figure that out for altendky/ssst#35 too.  Ok, here's our example showing it working in a `src/` package.

https://github.com/twisted/twisted/blob/7cf6c8bc320ac5fd96b4784f6feb932ea819856d/setup.cfg#L3
```python
version = attr: twisted.__version__
```